modules/sdk/gradle-plugins-defaults/CHANGELOG.markdown

```diff
@@ -380,12 +380,26 @@ following system properties to `true`:
 ### Changed
 - [LPS-68334]: Update the [Liferay Gradle Plugins] dependency to version 2.0.50.
 
+## 1.2.44 - 2016-10-06
+
+### Changed
+- [LPS-66396]: Update the [Liferay Gradle Plugins] dependency to version 3.0.0.
+
+### Fixed
+- [LPS-66396]: Remove dependency from Java 8 class.
```

Comment:
It was a wrong import in a few classes: [`java.io.UncheckedIOException`](https://docs.oracle.com/javase/8/docs/api/java/io/UncheckedIOException.html) instead of [`org.gradle.api.UncheckedIOException`](https://docs.gradle.org/current/javadoc/org/gradle/api/UncheckedIOException.html).